source/client-side-encryption/client-side-encryption.rst

```diff
@@ -294,6 +325,7 @@ MongoClient Changes
       bypassAutoEncryption: Optional<Boolean>; // Default false.
       extraOptions: Optional<Map<String, Value>>;
       tlsOptions: Optional<Map<String, TLSOptions>>; // Maps KMS provider to TLS options.
+      encryptedFieldsMap: Optional<Map<String, Document>>; // Maps namespace to encryptedFields.
```

Comment:
Sorry for the extra comment – in the scope this is called `encryptedFieldConfigMap`. I do like thie variant a bit more, but either way, once this spec PR is merged, we can rely on this name not being changed, right?

Reply:
Perfect, thank you!